int8 stop_play
---
string result
string fail_reason
